assets/queries/terraform/alicloud/oss_bucket_has_static_website/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "2b13c6ff-b87a-484d-86fd-21ef6e97d426",
+  "queryName": "OSS Bucket Has Static Website",
+  "severity": "HIGH",
+  "category": "Insecure Configurations",
+  "descriptionText": "Checks if any static websties are hosted on buckets",
```

Comment:
We should improve the query description for this query and `kics\assets\queries\terraform\aws\s3_static_website_host_enabled\test\positive_expected_result.json`